ports/servoshell/prefs.rs

```diff
@@ -616,7 +616,31 @@ pub(crate) fn parse_command_line_arguments(args: Vec<String>) -> ArgumentParsing
     let cmd_args = match cmd_args {
         Ok(cmd_args) => cmd_args,
         Err(error) => {
-            error.print_message(80);
+            // Because this is very early in startup stdout/stderr forward
+            // facilities might not be available yet
+            if cfg!(target_os = "android") || cfg!(target_env = "ohos") {
+                #[cfg(target_env = "ohos")]
+                {
+                    let mut builder = hilog::Builder::new();
+                    builder.set_domain(hilog::LogDomain::new(0xE0C3));
```

Comment:
This presumably has no effect without calling `builder.init()`.
It also looks like we initialize the hilog logger at the very beginning, so presumably we don't actually need to do anything here.